app/controllers/flowcommerce_spree/webhooks_controller.rb

```diff
@@ -21,5 +24,14 @@ def handle_flow_web_hook_event
                         end
       render json: result.except(:backtrace), status: response_status
     end
+
+    private
+
+    def check_organization
+      org = params[:organization]
+      return {} if org == FlowcommerceSpree::ORGANIZATION
```

Comment:
In case the organization does not match, should we log the error? I mean, from Flow side we cannot see these errors. so maybe it would be useful for troubleshooting purposes from our to have it??

Reply:
The error result is rendered as json, @sebastiandl  - it will be displayed in the logs.